modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1268,7 +1268,9 @@
     <string name="settings_use_dark_up_next_details">When enabled the Up Next will always use the dark theme, or will match the current theme when disabled</string>
     <string name="settings_use_dynamic_colors_widget">Use dynamic colors for widget</string>
     <string name="settings_use_dynamic_colors_widget_details">When enabled, widget colors will change based on the device theme colors.</string>
-    <string name="settings_use_embedded_podcast_artwork">Use embedded podcast artwork</string>
+    <string name="settings_use_embedded_podcast_artwork">Use embedded MP3 artwork</string>
```

Comment:
I'm not sure if `MP3 artwork` is the best term here. I'd prefer `file artwork` because we support other audio types as well and they might have embedded artworks as well.